### DISCUSSION TITLE
Need help with frontend, backend, and middle

### DISCUSSION THREAD

**Discussion by Apexdestroyer13** [Budibase/budibase#1353] [Help] [answered]
Project/File Index with geocoded map search

I have been searching for a template at least close to what I need. I haven't been able to find anything.
It would be for project/document management but not in the sense of tracking a projects progress.
I'm a land surveyor and I make plats (maps) of properties. I need to be able to find my previous maps within a Google Maps map.
Attached is my desired workflow for creating a new "job" and when complete, adding the additional details to the DB.
Also attached is the search app workflow I am looking for.

My digital files are growing quickly (about 5k pdf plats now) and I need to get this organized before it get out of control. I am currently using Google Forms, Google Sheets, and OneDrive but I need something better. I also have about 60k+ paper plats (most drawn and written by hand) that need to be scanned and indexed but that is another issue in its own that I will save for next time.

Also, If anyone has any experience creating apps like these, I could use a shove in the right direction. Thanks

[Completed Job Flow.pdf](https://github.com/Budibase/budibase/files/6247373/Completed.Job.Flow.pdf)
[New Job Flow.pdf](https://github.com/Budibase/budibase/files/6247374/New.Job.Flow.pdf)

**Comment by joebudi** [comment, marked as answer]
Hey! Apologies, for the late reply. Did you make any progress on this?

If you like, I am happy to jump on a call with you and discuss it when you are ready:
https://calendly.com/budibase/

**Reply by Apexdestroyer13** [reply]
Unfortunately, every solution I have found falls short. Retool seemed to work great with searching and locating points on a map but it lacked a frontend to create new entries.

**Reply by joebudi** [reply]
Hey! I am really sorry to hear that. We don't have a Google map component at the moment, but you can embed Google Maps using the embed component.

Also, currently there is no way to create a PDF. This is another popular request and will come in the next few months. I have went ahead and created an issue. You can following this here:
#1372
